pycqed/measurement/calibration_points.py

```diff
@@ -19,10 +19,12 @@ def __init__(self, qb_names, states, **kwargs):
         self.states = states
         default_map = dict(g=['I '], e=["X180 "], f=['X180 ', "X180_ef "])
         self.pulse_label_map = kwargs.get("pulse_label_map", default_map)
+        self.prep_params = None
 
     def create_segments(self, operation_dict, pulse_modifs=dict(),
                         **prep_params):
         segments = []
+        self.prep_params = prep_params
```

Comment:
same comment as above, I'm not sure I understand why we should save the prep_params as part of the state of the calibration_point. to me, those are two different things and the prep_params are just needed to create the segments and get the right indices, it could end up being odd if that changes the cal_point_object depending on the preparation parameters when creating segments, no?